libs/datamodel/connectors/sql-datamodel-connector/src/postgres_datamodel_connector.rs

```diff
@@ -1,10 +1,37 @@
+use datamodel_connector::error::ConnectorError;
 use datamodel_connector::{
     scalars::ScalarType, Connector, ConnectorCapability, NativeTypeConstructor, NativeTypeInstance,
 };
 use native_types::{NativeType, PostgresType};
+use tracing::log::kv::Source;
 
+const SMALL_INT_TYPE_NAME: &str = "SmallInt";
+const INTEGER_TYPE_NAME: &str = "Integer";
 const BIG_INT_TYPE_NAME: &str = "BigInt";
+// const DECIMAL_TYPE_NAME: &str = "Decimal";
+// const NUMERIC_TYPE_NAME: &str = "Numeric";
+const REAL_TYPE_NAME: &str = "Real";
+const DOUBLE_PRECISION_TYPE_NAME: &str = "DoublePrecision";
+const SMALL_SERIAL_TYPE_NAME: &str = "SmallSerial";
+const SERIAL_TYPE_NAME: &str = "Serial";
+const BIG_SERIAL_TYPE_NAME: &str = "BigSerial";
 const VARCHAR_TYPE_NAME: &str = "VarChar";
+const CHAR_TYPE_NAME: &str = "Char";
+const TEXT_TYPE_NAME: &str = "Text";
+// const BYTE_A_TYPE_NAME: &str = "ByteA";
+const TIMESTAMP_TYPE_NAME: &str = "Timestamp";
+const TIMESTAMP_WITH_TIMEZONE_TYPE_NAME: &str = "TimestampWithTimeZone";
```

Comment:
One blocker we have for adopting Prisma for all our data needs is the lack of support for native datatypes. Thanks for working on this.

Specifically, we have a column that has a `tstzrange` "range for timestamps with timezone": https://www.postgresql.org/docs/12/rangetypes.html#RANGETYPES-BUILTIN

The following suggestions would add this to these native datatypes. Is it possible to support this?

```suggestion
const TIMESTAMP_WITH_TIMEZONE_TYPE_NAME: &str = "TimestampWithTimeZone";
const TIMESTAMP_WITH_TIMEZONE_RANGE_TYPE_NAME: &str = "TimestampWithTimeZoneRange";
```